### ISSUE TITLE
Upgrade Classic McEliece to NIST round 4 specification

### ISSUE THREAD

**Issue by faern** [open-quantum-safe/liboqs#1314]
Hi! Are there any plans to upgrade the Classic McEliece KEM to the [round 4 specification](https://classic.mceliece.org/nist.html)?

The Rust implementation `classic-mceliece-rust` is doing that here: https://github.com/Colfenor/classic-mceliece-rust/pull/33

**Comment by baentsch** [comment]
Yes, round 4 algorithm updates are planned to be included in the next major release of `liboqs` and the downstream OQS projects. For McEliece, we normally import the code from [PQClean](https://github.com/PQClean/PQClean), so it may be good to ask the question also in that project.

**Comment by faern** [comment]
Great. Thanks. Submitted: https://github.com/PQClean/PQClean/issues/459

**Comment by thomwiggers** [comment]
https://github.com/PQClean/PQClean/pull/460 is ready for review.
